### PR TITLE
Added Code of Conduct and Fixed an installion Method

### PR DESCRIPTION
**Pull request by mraza007**
I have added the code of conduct make sure to add the contact email before you merge. I have also fixed API installation method.

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,54 @@
+# Code of Conduct
```
Comment (Huddie) [inline CONTRIBUTING.md]: Why is this file the same as the Code of Conduct?

File: CodeOfConduct.md [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+when an individual is representing the project or its community. Examples of
+representing a project or community include using an official project e-mail
+address, posting via an official social media account, or acting as an appointed
+representative at an online or offline event. Representation of a project may be
+further defined and clarified by project maintainers.
+
+## Enforcement
+
+Instances of abusive, harassing, or otherwise unacceptable behavior may be
+reported by contacting the project team at `<Add_EMAIL>` All
```
Comment (Huddie) [inline CodeOfConduct.md]: email should be: cunybot1@gmail.com

File: README.md [hunk 3]
```diff
@@ -38,5 +38,8 @@ Okay so you know what we're looking to do. Here's how you can help:
 3. git clone ....... from your fork
 
 #### Installing the package locally for testing
-To install and run this package locally, clone the repository and inside the root directory run `python3 -m pip install --user -e .`
+To install and run this package locally, clone the repository and inside the root directory run `python3 setup.py install --user`
```
Comment (Huddie) [inline README.md:41]: This has been tested?

**Review by Huddie — CHANGES_REQUESTED** [review]
Thanks for the start! I think you mistakenly pushed CONTRIBUTING.md with Code of Conduct text in it. Do you want to write up the CONTRIBTUING.md guide?  I think we prefer a SETUP.md and PULL_REQUEST_GUIDE.md split up getting started + making the contribution

File: README.md (hunk 3 above)
Comment (mraza007) [reply README.md:41]: Yup

Comment (Huddie) [reply README.md:41]: Perfect!

File: CONTRIBUTING.md (hunk 1 above)
Comment (mraza007) [reply CONTRIBUTING.md]: Since you opened an issue asking to add code of conduct

Comment (Huddie) [reply CONTRIBUTING.md]: But this filename is CONTRIBUTING.md 
You created 2 files CONTRIBUTING.md and CODE_OF_CONDUCT.md both with the same text inside. 

The CODE_OF_CONDUCT.md file looks great (except it needs you to fill in the proper email). Whats the purpose of the CONTRIBUTING.md file? Why does it have the same text?

File: CodeOfConduct.md (hunk 2 above)
Comment (mraza007) [reply CodeOfConduct.md]: I think I added the email

File: CONTRIBUTING.md (hunk 1 above)
Comment (mraza007) [reply CONTRIBUTING.md]: I was going to write a contributing.md but then realized it was supposed to be Code of Conduct I'll remove that file

File: CodeOfConduct.md (hunk 2 above)
Comment (Huddie) [reply CodeOfConduct.md]: You added it to CONTRIBUTING.md

File: CONTRIBUTING.md (hunk 1 above)
Comment (mraza007) [reply CONTRIBUTING.md]: Nvm I fixed it

**Comment by Huddie** [comment]
@mraza007 Everything looks good! Mind if we name it CODE_OF_CONDUCT.md rather than CodeOfConduct.md?

Don't worry about Bitrise failing, that has to do with something I changed on bitrise

**Comment by mraza007** [comment]
Fixing that lol
My git-workflow is hard to follow I fix up the name
I just need to understand what python api is doing

**Review by Huddie — APPROVED** [review]
thumbs up from me.
